docs/source/ttnn/ttnn/api.rst

```diff
@@ -440,6 +440,17 @@ Normalization
    ttnn.layer_norm
    ttnn.rms_norm
 
+
+Moreh Operations
+================
+
+.. autosummary::
+   :toctree: api
+   :nosignatures:
+   :template: function.rst
+
+   ttnn.moreh_sum
```

Comment:
```suggestion
   ttnn.operations.moreh.sum
```
In the case of `ttnn.rms_norm`, it can be called from Python as `ttnn.rms_norm`. However, `moreh_sum` cannot be called as `ttnn.moreh_sum`. This part should be adjusted to allow callable access.

Reply:
Is it? As far as I can see in the code, it is bound to the top level `ttnn` module, so it is in fact available on `ttnn.moreh_sum`

Reply:
If `ttnn.moreh_sum` is callable, then there's no need to modify it. 

However, From the unit tests, it seemed like `ttnn.operations.moreh.sum` was being called, leading to the assumption that `ttnn.moreh_sum` was not callable. 
ex)
```Python
 tt_output = ttnn.operations.moreh.sum(
```
https://github.com/tenstorrent/tt-metal/blob/main/tests/ttnn/unit_tests/operations/test_moreh_sum.py#L532

Could you please confirm that `ttnn.moreh_sum` is callable?

Reply:
It is callable due to the way it is bind to Python:
- Pybind11 generates `_ttnn.so` which is then exposed to Python as `_ttnn.operations.moreh.moreh_sum`
- `moreh.py` then takes this and re-bind it as `ttnn.operations.moreh.sum`
- TT's own `bind_registered_operation` binds the function to the top-level `ttnn` package, making it available as `ttnn.moreh_sum`

Reply:
I didn't fully understand the details. 
In any case, does this mean that it is possible to call it with `ttnn.moreh_sum`?

Reply:
Yes, it is possible to directly call `ttnn.moreh_sum`